articles/quickstart/spa/_includes/_authz_handle_scopes.md

```diff
@@ -0,0 +1,3 @@
+## Handle Scopes in the ${service} service
+
+Adjust your ${service} service to use a local member with any scopes you want to request when users log in. Use this member in your instance of the `auth0.WebAuth` object.
```

Comment:
As discussed on slack, this is a placeholder that is replaced when the snippet is inserted into an article (the variable is defined in the insert command).